chronologic/MyCrypto
# [WIP] [FEATURE] Schedule ERC20 tokens. Basic functionality.

### Description

Enables scheduling ERC20 tokens. User needs to do 2 transactions. First one - scheduling `transferFrom` function. Second is calling `approve` with scheduled transaction address as `_spender`.

### Changes

- Enable scheduling ERC20 tokens
- Triggers gas estimation after toggling `Send Later` functionality
- Simplifies `isSupportedUnit` function
- Removes `SetCurrentSchedulingToggle` type, as it isn't used
- Adds `AwaitingMIning` modal
- After mining of scheduling transaction automatically pre-fills approve transaction for user

### Steps to Test

1. Choose token as unit instead of ETH
2. Click Send Later
3. Fill transaction details
4. Wait for mining
5. Send approve transaction
4. Go to Chronos and see transaction has been scheduled, is a token transfer and there's no need to approve token transfer there

### Video
https://youtu.be/Y54eWusVARM
